lib/theme.dart

```diff
@@ -0,0 +1,44 @@
+import 'package:flutter/material.dart';
+
+class QlinkTheme {
+  static ThemeData get theme {
+    final themeData = ThemeData.dark();
+    final bodyText2 = themeData.textTheme.bodyText2?.copyWith(decorationColor: Colors.transparent);
+
+    final themeColor = Colors.indigo;
+    final textTheme = themeData.textTheme.copyWith(bodyText2: bodyText2);
+
+    return themeData.copyWith(
+      primaryColor: Colors.grey[800],
+      accentColor: themeColor[300],
+      toggleableActiveColor: themeColor[300],
+      textTheme: textTheme,
+      textSelectionTheme: themeData.textSelectionTheme.copyWith(
+        selectionColor: themeColor[100],
+        cursorColor: themeColor[300],
+      ),
+      snackBarTheme: SnackBarThemeData(
+        backgroundColor: themeData.dialogBackgroundColor,
+        contentTextStyle: bodyText2,
+        actionTextColor: themeColor[300],
+      ),
+
+      // Button styles
+      floatingActionButtonTheme: FloatingActionButtonThemeData(backgroundColor: themeColor[300]),
```

Comment:
I hate how the basic api for colors leaves unreadable numbers like `themeColor[300]` for which you have no idea what the color is just by reading the code.

is there a way to make the theme more readable? either constants, or by button the hex value in a comment or something?

Reply:
What IDE do you use? VSCode should show colors everywhere there is `Colors.something`.

![image](https://user-images.githubusercontent.com/9084735/110567584-c1c7d400-811f-11eb-979d-b2008b30d270.png)

Reply:
I'm using a JetBrains IDE (not Android Studio).

For both VSCode and IntelliJ, it looks like the colors are showing up for lines 8 and 12, but not for the lines with `themeColor[SOME_NUMBER]`


_good_

<img width="947" alt="Screen Shot 2021-03-09 at 21 40 34" src="https://user-images.githubusercontent.com/25187726/110567923-3d298580-8120-11eb-8148-f116f0cfefa7.png">

_sad_

<img width="754" alt="Screen Shot 2021-03-09 at 21 40 45" src="https://user-images.githubusercontent.com/25187726/110567924-3d298580-8120-11eb-982d-89fcb48aafbd.png">

Reply:
Oh that is a problem. I wonder if I need to assign it to something other than `final`.

Reply:
maybe we need to open a PR against the flutter plugin for our respective IDEs...

Reply:
Something so basic can't have been overlooked, right? Surely I'm doing something wrong?

Reply:
I've just added it to the kanbad board for now.  https://github.com/qlink-2/qlink/projects/1#card-56567051

We can promote it to an issue in the future, maybe.

Reply:
If not overlooked, at least not yet solved.

Here's the internal definition of a color in `Colors`. Notice one of the colors doesn't render in my IDE:

<img width="634" alt="Screen Shot 2021-03-09 at 22 02 56" src="https://user-images.githubusercontent.com/25187726/110570018-4cf69900-8123-11eb-8644-2b9abd58ea04.png">

Reply:
Found an issue: https://github.com/flutter/flutter-intellij/issues/4079

The way they show colors is currently "dumb".

Reply:
Wow and it's the primary value too 🤢

And here I was thinking Flutter had mature tooling

Reply:
It's a community effort, for better and worse